en/components/chip.md

```diff
@@ -0,0 +1,182 @@
+        <span #label [class]="'igx-chip__text'">{{chip.text}}</span>
+    </igx-chip>
+</igx-chips-area>
+```
+
+```ts
+public ngOnInit() {
+    chipArea.forEach((chip) => {
+        chip.draggable = true;
+    });
+}
+
+public chipsOrderChanged(event) {
+    const newChipList = [];
+    for (const chip of event.chipsArray) {
+        const chipItem = this.chipList.filter((item) => {
+            return item.id === chip.id;
+        })[0];
+        newChipList.push(chipItem);
+    }
+    this.chipList = newChipList;
+    event.isValid = true;
+}
+
+```
+
+![Dragging](../images/dragging.gif)
+
+#### Chip Templates
+
+The `IgxChipComponent`'s main structure consists of chip content, `remove button`, `prefix`, `suffix` and `connector`. All of those elements are templatable except the `remove button`.
+
+The content of the chip is taken by the content defined inside the chip template except elements that define the `prefix`, `suffix` or `connector` of the chip. You can define any type of content you need.
+
+The `prefix` and `suffix` are also elements inside the actual chip area where they can be templated by your preference. The way they can be specified is by respectively using the `IgxPrefix` and `IxgSuffix` directives.
+
+Example of using an icon for `prefix`, a text for `label` and a custom icon button for `suffix`:
+
+```html
+<igx-chip *ngFor="let chip of chipList" [id]="chip.id">
+    <igx-icon igxPrefix fontSet="material" [name]="'drag_indicator'"></igx-icon>
+    <span #label [class]="'igx-chip__text'">{{chip.text}}</span>
+    <span igxSuffix *ngIf="removable" igxButton="icon" (click)="onClick()">
+        <igx-icon fontSet="material" [name]="'close'"></igx-icon>
+    </span>
+</igx-chip>
+```
+
+The `connectors` of the `igxChip` are fully templatable and are positioned after each chip. Their purpose is to provide a way to link two chips next to each other with an icon/text or anything you would like to use. The last chip (most right) does not have connector applied. Connectors hide while dragging chips around and show again when interactions with the chips have finished. The connector is defined by using the `IgxConnector` directive.
+
+Example of using prefix connector:
+
+```html
+<igx-chip *ngFor="let chip of chipList" [id]="chip.id">
+    <span #label [class]="'igx-chip__text'">{{chip.text}}</span>
+    <span igxConnector> -> </span>
+</igx-chip>
+```
+
+Example of using suffix connector:
+
+```html
+<igx-chip *ngFor="let chip of chipList" [id]="chip.id">
+    <span #label [class]="'igx-chip__text'">{{chip.text}}</span>
+    <span igxSuffixConnector> -> </span>
+</igx-chip>
+```
+
+#### Keyboard Navigation
+
+The chip can be focused using the `Tab` key or by clicking on them. Chips can be reordered using keyboard navigation:
+
+- Keyboard controls when the chip is focused:
+
+  - <kbd>LEFT</kbd> - Focuses the chip on the left
+  - <kbd>RIGTH</kbd> - Focuses the chip on the right
```

Comment:
RIGHT?

Reply:
Yes, right. Sorry, my mistake.

Reply:
No worries, it's just something I noticed while translating the topic.